integrated_channels/blackboard/exporters/content_metadata.py

```diff
@@ -13,3 +13,19 @@ class BlackboardContentMetadataExporter(ContentMetadataExporter):
     """
         Blackboard implementation of ContentMetadataExporter.
     """
+    DATA_TRANSFORM_MAPPING = {
+        'name': 'title',
+        'externalId': 'key',
+        'description': 'description',
+        'courseId': 'key',
+    }
+
+    DESCRIPTION_TEXT_TEMPLATE = "<a href={enrollment_url}>Go to edX course page</a><br />"
+
+    def transform_description(self, content_metadata_item):
+        """
+        This will show a link to edX course on blackboard course description
+        """
+        enrollment_url = content_metadata_item.get('enrollment_url', None)
+        url_link = self.DESCRIPTION_TEXT_TEMPLATE.format(enrollment_url=enrollment_url)
+        return url_link
```

Comment:
wouldn't this transform the description into just a link?
Ie instead of "This course will teach you blah blah blah....[enroll here]" you get "[enroll here]"

Reply:
correct observation! Indeed not my fav solution as of now. The issue I found is that 'description' does not show up anywhere on course home page on blackboard. to make the course link findable in blackboard the only location I have found so far is the 'description' column shows in the search page. Hence this 'hack'

pro of approach: What this does do it nicely show a link when someone searches for a course. Hence helps with course discovery.
con: blackboard description is just a link

so perhaps I can come back and re-address this once I chat with Ken. For now, this seems to be the 'best' solution

Reply:
Hmmm I guess I'm still confused why it's only a link instead of the description + link? Assuming there is nowhere else that we'd use the description, why not put all of it there?

Otherwise, if the plan is for this to just be a link (whether it be for this PR or permanently) then you don't need to map it to the short_description field. Just map it to the enrollment url and then modify it slightly in your "transform_description" method.

Reply:
yeah putting description there right now messes up how it shows up on the course search page, however we could put a shorter description plus link there if we want. But yes if I wanted just a link there, I should map to the enrollment_url field instead of description, yes. let me try that change for a incremental improvement

Reply:
so that would mean just changing transform_description to transform_enrollment_url right?

like was thinking

```
DATA_TRANSFORM_MAPPING = {
...
        'description': 'enrollment_url',
...
    }
```

and 

```
    def transform_enrollment_url(self, content_metadata_item):

```

Reply:
made this change. still not feeling awesome of a solution, I will expect to improve on this as we continue our work with blackboard